InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -510,6 +514,33 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 
 		scene->scheduleUpdate();
 	}
+	else if (event->key() == Qt::Key_F12)
```

Comment:
F12 clashes with a button assigned by the debugger. Please don't use this. Some day I have to resolve all the shortcut mess that we're in, to make it easier to assign key shortcuts and keep track of what's already there.

For now, we should just find an used key combination, even if it's not that convenient. Let's go with CTRL+F12. Feel free to suggest something else, but do a quick text search over the entire Envision project to make sure that the key is free.